src/components/PollList/pollList.js

```diff
@@ -0,0 +1,104 @@
+import "./pollList.css";
+import { useState, useEffect } from "react";
+import { useNavigate } from "react-router-dom";
+import { useDispatch, useSelector } from "react-redux";
+import { Button, Form, Container, Navbar } from "react-bootstrap";
+import { Trash, PencilSquare } from "react-bootstrap-icons";
+import pollList from "../../redux/pollList/actions/pollList";
+import { removeUserData } from "../../redux/login/actions/login";
+
+const PollList = () => {
+  const dispatch = useDispatch();
+  const navigate = useNavigate();
+  const [pollIDs, setPollIDs] = useState([]);
+  const pollQuestion = useSelector((state) => state.pollList.pollList);
+  const userDetailsFromLocalStorage = JSON.parse(localStorage.getItem("user"));
+
+  useEffect(() => {
+    setPollIDs(JSON.parse(localStorage.getItem("pollIDs")) ?? []);
+    dispatch(pollList());
+  }, []);
+
+  const handleOptionClick = (pollID) => {
+    if (!pollIDs.includes(pollID)) {
+      setPollIDs((prevPollIDs) => [...prevPollIDs, pollID]);
+      localStorage.setItem("pollIDs", JSON.stringify([...pollIDs, pollID]));
+    }
+  };
+
+  return (
+    <div>
+      <Navbar className="navbar">
```

Comment:
Nav bar should be a common component, I should not be directly written inside poll list @PuniaMohit